integration-test/src/python/test_runner/main.py

```diff
@@ -88,6 +88,20 @@ def fetchResultFromServer(serverAddress, serverPort, topologyName):
       time.sleep(RETRY_INTERVAL)
 
   logging.error("Failed to fetch results after %d attempts" % RETRY_ATTEMPTS)
+  # For testing
+  # Cat the results
+  print "heron-executor.stdout"
+  cat_command = "cat /home/travis/.herondata/topologies/local/heron-staging/%s/heron-executor.stdout" %(topologyName)
```

Comment:
Can we make this relative to $HOME so it can also be tested locally? Also space after %, here and 2 more locations below.